src/panels/basic-info.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License.
+ */
+
+import { Button } from "../button";
+import { IPanel, IPanelConfig } from "../ipanel";
+import { PanelFrame } from "../panel-frame";
+
+export interface IBasicButton {
+    emoji: string;
+    title: string;
+    value(): string;
+}
+
+/** Describes the configuration options available for the network panel */
+export interface IBasicInfoPanelConfig extends IPanelConfig {
+    buttons?: IBasicButton[];
+    /** The name of the panel */
+    panelName?: string;
+}
+
+/** A set of default configuration options for the navigation timings panel */
+const basicInfoPanelDefaultConfig: Required<IBasicInfoPanelConfig> = {
+    panelName: "Basic Info",
+    buttons: [],
+};
+
+/**
+ * Provides a panel that shows the navigation timings for a page
+ */
+export class BasicInfoPanel implements IPanel {
+    /** The settings for this panel. */
+    private readonly config: Required<IBasicInfoPanelConfig>;
+
+    /** Construct the panel. */
+    public constructor(_frame: PanelFrame, config?: IBasicInfoPanelConfig) {
+        this.config = { ...basicInfoPanelDefaultConfig, ...config };
+    }
+
+    /** Get buttons. */
```

Comment:
Nit: this JSDoc doesn't really help the reader, and in fact makes this class just a bit harder to read for taking up more space. Consider clarifying it or getting rid of it, maybe?

Reply:
The tslint requires it, so I'll make it a little better.